tests/test_conversion.py

```diff
@@ -85,6 +89,13 @@ def _test_to_json(self, mdoc, test: SSSOMTestCase):
         with open(test.get_out_file("json"), "w") as file:
             write_json(msdf, file, serialisation="json")
 
+    def _test_to_fhir_json(self, mdoc, test: SSSOMTestCase):
```

Comment:
## FYI: Add FHIR test to `test_conversion.py`
Adding too much detail here feels redundant with the tests in `test_writers.py`. And the ontoportal format has both a `test_ontoportal_writer()` and `_test_to_ontoportal_json()` here, as well as another test in `writers.py`. There is some degree of arbitrariness as to which of these functions should have the most assertions. I've chosen to not construct a 2nd function in `test_conversion.py`, so...

...as of this PR I have added FHIR test hooks w/ minor assertions in two places:
- `test_cli.py`
- `test_conversion.py`

...and I've updated to put more assertions in:
- `test_writers.py`